luigi/execution_summary.py

```diff
@@ -89,34 +91,39 @@ def _get_str(task_dict, extra_indent):
     """
     This returns a string for each status
     """
+    summary_length = int(configuration.get_config().get('core', 'summary-length', 5))
```

Comment:
This is not how we do configuration in luigi. :)

http://luigi.readthedocs.org/en/latest/configuration.html#configuration-classes